client/js/components/fragments/MiniProfile.js

```diff
@@ -1,19 +1,32 @@
 import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
-import DefaultAvatar from '../../../img/avatar.jpg';
-import 'file?name=[name].[ext]!../../../img/avatar.jpg';
-
 class MiniProfile extends Component {
   render() {
     const { user } = this.props;
+
+    let avatar = null;
+    if (user.avatar) {
+      avatar = (
+        <span className="media-left">
+          <img className="media-object img-circle" src={user.avatar} alt={`${user.firstName} ${user.lastName}`}/>
+        </span>
+      );
+    }
+
+    let name = (<p className="text-muted">{user.email}</p>);
```

Comment:
Single line elements don't need to be wrapped in () braces - I think we don't use braces anywhere else but this place